portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+/** Models a task that the user creates. */
```

Comment:
Similarly, consider updating this comment so you can use it for both creation and import flows.

Reply:
Updated.